api/src/main/resources/workbench-api.yaml

```diff
@@ -804,6 +804,20 @@ paths:
           description: Workspace deletion request accepted
           schema:
             "$ref": "#/definitions/EmptyResponse"
+  "/v1/workspaces/{workspaceNamespace}/{workspaceId}/get-data-page-items":
+    parameters:
+      - "$ref": "#/parameters/workspaceNamespace"
+      - "$ref": "#/parameters/workspaceId"
+    get:
+      tags:
+      - workspaces
+      description: Gets all items shown on the data page of the workspcae
```

Comment:
This definition is from the perspective of an existing UI design or implementation. It should really be written in a stand-alone way, so that a developer of a future workbench client could use it. That is, the API definition shouldn't really be in terms of the UI.